src/controllers/invoices.js

```diff
@@ -114,11 +116,17 @@ export default function () {
             </tr>
           </thead>
           <tbody>
-            ${(order.items
-              ? order.items.filter(item => item.quantity > 0).reduce((trsStr, item, i) => trsStr + `
+            ${(items
+              ? items.filter(item => item.quantity > 0).reduce((trsStr, item, i) => trsStr + `
                 <tr>
                   <td>${(i + 1)}</td>
-                  <td>${item.name} (${item.sku})</td>
+                  <td>${item.name} (${item.sku})
+                  ${item.customizations
+                    ? item.customizations.reduce((trs, custom, index) => trs + `
+                    <br><em>${custom.label}</em>:<mark>${custom.option.text}</mark>`
```

Comment:
```suggestion
                    ? item.customizations.reduce((trs, custom, index) => trs +
                    `<br><em>${custom.label}</em>:<mark>${(custom.option ? custom.option.text : '')}</mark>`
```

`custom.option` pode ser undefined.